sdcm/sct_events.py

```diff
@@ -877,31 +871,11 @@ def __eq__(self, other):
         return True
 
 
-class TestKiller(Process):
-    __test__ = False  # Mark this class to be not collected by pytest.
-
-    def __init__(self, timeout_before_kill=2, test_callback=None):
-        super(TestKiller, self).__init__()
-        self._test_pid = os.getpid()
-        self.test_callback = test_callback
-        self.timeout_before_kill = timeout_before_kill
-
-    def run(self):
-        for event_type, message_data in EVENTS_PROCESSES['MainDevice'].subscribe_events():
-            if event_type == 'KillTestEvent':
-                time.sleep(self.timeout_before_kill)
-                LOGGER.debug("Killing the test")
-                if callable(self.test_callback):
-                    self.test_callback(message_data)
-                    continue
-                if not safe_kill(self._test_pid, signal.SIGTERM) or pid_exists(self._test_pid):
-                    safe_kill(self._test_pid, signal.SIGKILL)
-
-
-class EventsFileLogger(Process):  # pylint: disable=too-many-instance-attributes
+class EventsFileLogger(multiprocessing.Process):  # pylint: disable=too-many-instance-attributes
     def __init__(self, log_dir):
         super(EventsFileLogger, self).__init__()
         self._test_pid = os.getpid()
+        self._stop_event = multiprocessing.Event()
```

Comment:
It'll be better to have this property named without leading underscore like we have in other classes. Just to be consistent.